pkg/collect/host_filesystem_performance_test.go

```diff
@@ -57,3 +60,130 @@ func TestGetPercentileIndex(t *testing.T) {
 		})
 	}
 }
+
+func Test_parseCollectorOptions(t *testing.T) {
+	type args struct {
+		hostCollector *troubleshootv1beta2.FilesystemPerformance
+	}
+	tests := []struct {
+		name        string
+		args        args
+		wantCommand []string
+		wantOptions *FioJobOptions
+		wantErr     bool
+	}{
+		{
+			name: "Happy spec",
+			args: args{
+				hostCollector: &troubleshootv1beta2.FilesystemPerformance{
+					HostCollectorMeta: troubleshootv1beta2.HostCollectorMeta{
+						CollectorName: "fsperf",
+					},
+					OperationSizeBytes:          1024,
+					Directory:                   "/var/lib/etcd",
+					FileSize:                    "22Mi",
+					Sync:                        true,
+					Datasync:                    true,
+					Timeout:                     "120",
+					EnableBackgroundIOPS:        true,
+					BackgroundIOPSWarmupSeconds: 10,
+					BackgroundWriteIOPS:         100,
+					BackgroundReadIOPS:          100,
+					BackgroundWriteIOPSJobs:     1,
+					BackgroundReadIOPSJobs:      1,
+				},
+			},
+			wantCommand: []string{
+				"fio",
+				"--name=fsperf",
+				"--bs=1024",
+				"--directory=/var/lib/etcd",
+				"--rw=write",
+				"--ioengine=sync",
+				"--fdatasync=1",
+				"--size=23068672",
+				"--runtime=120",
+				"--output-format=json",
+			},
+			wantOptions: &FioJobOptions{
+				RW:        "write",
+				IOEngine:  "sync",
+				FDataSync: "1",
+				Directory: "/var/lib/etcd",
+				Size:      "23068672",
+				BS:        "1024",
+				Name:      "fsperf",
+				RunTime:   "120",
```

Comment:
I'm guessing we are happy with `120s` maximum runtime, right?

Reply:
@banjoh it keeps the same max time as the previous implementation, I'm OK with it.  `fio` usually finishes in less time than that, but allows for a max runtime of 2 minutes.